### ISSUE TITLE
Implement A Functionality To Open This Program In Background

### ISSUE THREAD

**Issue by hetulbhatt** [hetulbhatt/KeyFlow#14]
Title says it all.

**Comment by ioki9** [comment]
hey, in background, do you mean in a system tray or just hidden? And if just hidden, how do you expect user to shut it down, except ending the process in a task manager?

**Comment by ioki9** [comment]
hey, I think I can do it if you willing to assign me to it.

**Comment by hetulbhatt** [comment]
Yes, feel free to work on this task. Let me know if you want to discuss the approach.
